web/controllers/page_controller.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Apr.PageController do
   use Apr.Web, :controller
 
+  plug BasicAuth, realm: "Admin Area", username: "admin", password: "secret"
```

Comment:
@ashkan18 why the hardcoded values `username: "admin", password: "secret"` here?